MapDiffBot/MapDiffBot.csproj

```diff
@@ -18,7 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Cyberboss.AspNetCore.AsyncInitializer" Version="1.2.0" />
-    <PackageReference Include="GitHubJwt" Version="0.0.3" />
+    <PackageReference Include="GitHubJwt" Version="0.0.4" />

```

Comment:
Is this literally all thats needed?

Reply:
This isn't the fix but it's good regardless